src/styles/variable.scss

```
@@ -0,0 +1,10 @@
+$card-width: 200px;
+$card-margin: 48px;
+$inside-padding: 26px;
+$catalog-font-family: roboto, verdana, geneva, tahoma, sans-serif;
```

you have some issues with tests, one of the reason is may be wrong fonts. Try it 
```suggestion
$catalog-font-family: "Roboto", sans-serif;
```